packages/gestalt/src/Popover/InternalPopover.jsdom.test.js

```diff
@@ -0,0 +1,29 @@
+// @flow strict
```

Comment:
Cos we have Popover and PopoverEducational tests, we can leave this test or remove as it not adding much value as it's redundant with the other two